### ISSUE TITLE
Mappings from file?

### ISSUE THREAD

**Issue by rkosegi** [mojohaus/rpm-maven-plugin#114]
It would be really nice if mappings can be defined in XML file.
For example it could be constructed by some means during generate-resources phase.
One particular use case is when you have lots of structurally distinct rpms as child modules but you can figure out structure during build (eg. by presence of some directories). Then mappings can be constructed (eg. by inlined groovy script in parent pom ) and plugin configuration in children will remain minimal.

**Comment by Stockatansky** [comment]
I like this idea. Now our pom is exploded with rpm mapping. Is already some started with this our do we need to writer it our self?

**Comment by robth** [comment]
Agree. This would be very useful. Would submitting a PR have a chance of being accepted? As in: do the maintainers agree with this idea?
